src/lib/djerba/plugins/fusion/preprocess.py

```diff
@@ -256,17 +257,6 @@ def drop_duplicates_merge_columns(self, df):
     
     def fix_reading_frames(self, df):
         """
-        Makes two columns: one with all reading frames (informative), and one that just says "Multiple Frames" if there are multiple.
-        Valid reading frames are in-frame, out-of-frame, and stop-codon.
-        According to the arriba documentation:
-        
-            reading_frame : This column states whether the gene at the 3' end of the fusion is fused in-frame or out-of-frame. 
-            The value stop-codon indicates that there is a stop codon prior to the fusion junction, such that the 3' end is not translated, 
-            even if the reading frame is preserved across the junction. The prediction of the reading frame builds on the prediction of the 
-            peptide sequence. A dot (.) indicates that the peptide sequence cannot be predicted, for example, because the transcript sequence 
-            could not be determined or because the breakpoint of the 5' gene does not overlap a coding region
-            
-        Anything else should be left as nan.
```

Comment:
Why remove this explanation? Would be helpful to know why the dot (.) is significant. Otherwise all we have to go by is the rather vague function name "fix_reading_frames".
Not necessary to reinstate the whole thing, but a brief comment would be appropriate.